tasks/files-to-granules/schemas/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "FilesToGranulesConfig",
+  "description": "Describes the config used by the files-to-granules task",
+  "type": "object",
+  "required": [
+    "input_granules"
+  ],
+  "properties": {
+    "input_granules": {
+      "type": "array",
```

Comment:
This says it's an array but, in the files-to-granules task, it [defaults to an empty object](https://github.com/nasa/cumulus/blob/1e91db0aa4af896f3d0911e97753a4fe3e1b9d68/tasks/files-to-granules/index.js#L76).